### ISSUE TITLE
There's differences between the Dashboards in the app and the Dashboard in the website 

### ISSUE THREAD

**Issue by shiramax** [matobet/moVirt#210]
in cpu : 
the app shows Allocated 100% in the website 22%
the app shows Allocated 4% in the website 6%

in Memory : 
the app shows Allocated 39%, in the website 134%
the app shows over Commit  12% , in the website 40%

in Storage : 
the app shows Allocated 0% in the website 6%
the app shows over Commit  0% , in the website 1%

the app: 

the website:
![image](https://cloud.githubusercontent.com/assets/5609929/19849115/f668340e-9f59-11e6-8014-01755bbaec80.png)

the app: 
![image](https://cloud.githubusercontent.com/assets/5609929/19849162/49303fc4-9f5a-11e6-9df4-0f4a260662da.png)

**Comment by jelkosz** [comment]
@shiramax yes, that is true. The reason is that the webadmin is using an average for some longer time (which is not exposed using REST API) while moVirt is using the current values.

**Comment by shiramax** [comment]
if there are difference between the webadmin and the app, it should reflected , 
otherwise it seems like a bug...

**Comment by suomiy** [comment]
@shiramax you are right, I discovered three separated bugs because of that
- missing disks on v4 api #229
- over commit is incorrect in moVirt #230
- bug in Dashboard in Open Virtualization Manager: available/used resources in CPU/Memory are incorrect - to be tackled

**Comment by suomiy** [comment]
There should be some kind of a description of following behaviour here on GH.

proposal:

**Dashboard description**

Physical view
   - displays utilizations for hosts (CPU, Memory) and storage domains (Storage) 
        
Virtual view
  - displays utilizations for vms (CPU, Memory) and vms' disks (Storage)    

Over commit (Physical, Virtual)
   - over commit displays ratio (in percentages)  of running virtual resources in relation to the total of physical resources
   - over commit's allocated displays ratio (in percentages) of allocated virtual resources in relation to the total of physical resources 
    
**Differences in view compared Open Virtualization Manager**

All dashboard
  - moVirt uses recently fetched statistics vs Open Virtualization Manager which uses average over last 5 minutes
        
Virtual view and over commit
  - may be different if there is limitation (max count or query) set for fetched vms

**Comment by suomiy** [comment]
please write me, if something is missing in this description

@jelkosz @matobet  it would be probably useful to start a wiki. The readme is getting quite long.

**Comment by suomiy** [comment]
@shiramax  btw next time you can open the dashboard on tablet to see the full view and if the number of  Virtual Machines is the same as in moVirt you do not have to worry about this 

> Virtual view and over commit
> may be different if there is limitation (max count or query) set for fetched vms

\+ also full view has more information for bug screenshots

**Comment by shiramax** [comment]
@suomiy , i have few  question about the Dashboard: 
The dasborad titles look like this : 
" Allocated 
**X Availble of X GIB**
Over commit 
 **2% (allocated 13%) "** 

- what's the difference between the two allocated? 
- Is the overcommit should be like the webadmin? 
- "bug in Dashboard in Open Virtualization Manager: available/used resources in CPU/Memory are incorrect - to be tackled"-> do you have the bug id?

**Comment by suomiy** [comment]
@shiramax 

> what's the difference between the two allocated?
 - allocated designates allocation of Physical/Virtual resource
 - over commit allocated should designate maximum over commit allocation (that means e.g.  total memory of all vms, which can be higher than actual physical resource ) 
- over commit used (2%) is e.g. used memory/cpu of running vms 

- over commit in storage is used/allocated size of all disks in the datacenter 
> Is the overcommit should be like the webadmin?

 it should be after fixing #230, if you turn off filtering 
>do you have the bug id?

https://bugzilla.redhat.com/show_bug.cgi?id=1401010

**Comment by suomiy** [comment]
> over commit in storage is used/allocated size of all disks in the datacenter

this should be also in Documentation, that storage overcommit/virtual will not depend on number of vms returned

**Comment by shiramax** [comment]
I find the UI of the Dashboard problematic,

the issues i have with current app dashboard : 

- In ovirt engine the UI is more simple :
![image](https://cloud.githubusercontent.com/assets/5609929/20925790/8e98dbce-bbc1-11e6-9f0a-8f7afe5fb1d8.png)
 In the app it's confusing for me to understand this two 'allocated' parameters because they used in the same word).  
- The CPU don't have the X Available of X GIB in the app (In the admin portal it's exists) 
- The parameters of "X Available of X GIB" - can be removed, because they are already inside the circle. 
- The VIRTUAL DashBoard is not necessary because it's not interesting, it's not in the webadmin why should i need it in the app? 

To sum up all my problems, I think that the Dashboard should be simple, I should be able to see in one dashboard the General status, and i should able to understand the calculations in the Dashboard.

**Comment by suomiy** [comment]
> In the app it's confusing for me to understand this two 'allocated' parameters because they used in the same word).

I guess you are right, we will remove the first allocated so it will be consistent with ovirt dashboard

> The CPU don't have the X Available of X GIB in the app (In the admin portal it's exists)

this will get reevaluated in over commit issue

> The parameters of "X Available of X GIB" - can be removed, because they are already inside the circle.

No, these are important information

> The VIRTUAL DashBoard is not necessary because it's not interesting, it's not in the webadmin why should i need it in the app?

to have better overview of the vms? ;)  Tackled in #230

> To sum up all my problems...

We will have wiki to describe the values. But the first priority of the dashboard should be to be informative

**Comment by shiramax** [comment]
@suomiy, 
The parameters of "X Available of X GIB" - can be removed, because they are already inside the circle.
No, these are important information

There are too many values that gives the same information but it confusing way.
99 Available of 100 GIB
and 
1% Used (inside the circle) 

in percents it's more easy to understand this calculation but in Storage and Memory, the Values are not presented in percents and this is confusing.

So I think it should be removed or have the same semantic. 
1 is Used of 100 GIB
and 
1% Used (inside the circle)

**Comment by suomiy** [comment]
@shiramax 
One thing is that we are using different units for cpu and memory/storage. So it is not a problem that these are different.

We could also show usage in percentages but this information is already there (not numerically, but the (green|orange|red) circle shows it)

And we also want to keep webadmin and movirt dashboard consistent
